vm/src/pyobject.rs

```diff
@@ -1259,6 +1259,26 @@ where
     }
 }
 
+pub trait IntoPyClass {
+    const NAME: &'static str;
+    const DOC: Option<&'static str> = None;
+
+    fn _extend_class(ctx: &PyContext, class: &PyClassRef);
```

Comment:
Renamed this function to not underscore, since this function must be implemented by trait implementors. Allthough this is rust, many coders will come from python background, where the single underscore prefix indicates some privateness.